### DISCUSSION TITLE
Missing Town Ranks

### DISCUSSION THREAD

**Discussion by TheSamles** [TownyAdvanced/Towny#5808] [Q&A] [answered]
For some reason, I am missing town ranks, such as co-mayor and many others. I only have a few such as an assistant, helper, and Sherriff. How do I get the rest?

**Comment by Warriorrrr** [comment, marked as answer]
The assistant, helper and sheriff ranks are default towny ranks, "co-mayor" is not a default towny rank, you have to add it manually.

**Reply by TheSamles** [reply]
Thank you. Is this done in the config or on a per town basis?

**Reply by LlmDl** [reply]
In your townyperms.yml file.
